wiki/Help_centre/Upgrading_to_lazer/vi.md

```diff
@@ -0,0 +1,288 @@
+| DirectX / Vulkan | ![Partial][partial][^compatibility-mode] | ![No][false][^coming-soon] |
+| Hỗ trợ điện thoại | ![No][false] | ![Yes][true] |
+| Kiến trúc đa luồng | ![No][false] | ![Yes][true] |
+| Video sử dụng tăng tốc phần cứng | ![No][false] | ![Yes][true] |
+| Chỉnh sửa giao diện người dùng | ![No][false] | ![Yes][true] |
+| Bộ quy tắc tùy chỉnh (các chế độ chơi) | ![No][false] | ![Partial][partial][^dll] |
+| Kho lưu trữ tệp trùng lặp | ![No][false] | ![Yes][true][^share-files] |
+| Điều chỉnh khu vực máy tính bảng | ![No][false] | ![Yes][true] |
+
+### Giao diện người dùng và tạo skin
+
+| Tính năng | ổn định | lazer |
+| :-- | :-- | :-- |
+| Hỗ trợ skin | ![Yes][true] | ![Partial][partial][^gameplay-only] |
+| Nhóm các bài hát theo các chế độ | ![Yes][true] | ![No][false] |
+| Skin trong trò chơi / Chỉnh sửa bố cục giao diện người dùng | ![No][false] | ![Yes][true] |
+
+### Gameplay và skin
+
+| Tính năng | ổn định | lazer |
+| :-- | :-- | :-- |
+| Hiển thị điểm hiệu suất chính xác | ![Partial][partial][^online] | ![Yes][true] |
+| Các mod cài đặt trước | ![No][false] | ![Yes][true] |
+| Các cài đặt mod | ![No][false] | ![Yes][true] |
+| Các mod "vui" | ![No][false] | ![Yes][true] |
+| Chuẩn hóa màu combo[^normalisation] | ![No][false] | ![Yes][true] |
+| Giữ để HUD | ![No][false] | ![Yes][true][^hold-for-hud] |
+| Căn chỉnh độ trễ | ![Partial][partial][^offset-calibration-stable] | ![Yes][true][^offset-calibration-lazer] |
+| Thanh trượt osu! "di chuyển như con rắn" khi đang kéo | ![No][false] | ![Yes][true][^can-disable] |
+| "Notelock" thân thiện với người dùng | ![No][false] | ![Yes][true][^note-lock] |
+| Tô màu các nốt osu!mania theo thời gian | ![No][false] | ![Yes][true] |
+
+### Các hệ thống online
+
+| Tính năng | ổn định | lazer |
+| :-- | :-- | :-- |
+| Nộp điểm số | ![Yes][true] | ![Partial][partial][^score-reset-balance] |
+| Bảng xếp hạng beatmap | ![Yes][true] | ![Partial][partial][^score-reset-isolated] |
+| Các thống kê hồ sơ | ![Yes][true] | ![Yes][true] |
+| Các huân chương | ![Yes][true] | ![No][false] |
+| Điểm hiệu suất | ![Yes][true] | ![Partial][partial][^score-reset-isolated] |
+| Trò chuyện thời gian thực | ![Partial][partial][^stable-chat] | ![Yes][true] |
+| Wiki / Tin tức / Nhật kí thay đổi / Xếp hạng | ![No][false] | ![Yes][true][^online-content] |
+| Hồ sơ người dùng | ![No][false] | ![Yes][true] |
+| Danh sách beatmap | ![Partial][partial][^direct-supporter] | ![Yes][true] |
+| Phòng nhiều người chơi không giới hạn | ![No][false][^multi-room-max] | ![Yes][true] |
+| Xem nhiều người  | ![No][false] | ![Yes][true] |
+| Đồng hồ đếm ngược | ![Partial][partial][^countdown-timers-stable] | ![Yes][true][^countdown-timers-lazer] |
+| Chế độ xếp hàng | ![No][false] | ![Yes][true][^queue-modes] |
+| Các lệnh nhiều người chơi | ![Yes][true] | ![No][false] |
+| Tag phối hợp | ![Yes][true] | ![No][false] |
+| Các danh sách phát (bảng xếp hạng do người dùng quản lý) | ![No][false] | ![Yes][true] |
```

Comment:
```suggestion
| Các playlist (bảng xếp hạng do người dùng quản lý) | ![No][false] | ![Yes][true] |
```
This is "playlist", formerly known as timeshift.
Not a set of songs that you pre-set it for the music player to play.